### PR TITLE
Align CDK Template

### PR DESCRIPTION
**Pull request by samchungy**
Ripped out changes from: https://github.com/seek-oss/skuba/pull/1518

### CODE REVIEW

**Review by 72636c — APPROVED** [review]
Just a few questions on the existing Serverless template, feel free to merge if we'd rather tackle later/elsewhere.

File: template/lambda-sqs-worker-cdk/infra/appStack.ts [hunk 1]
```diff
@@ -58,6 +60,31 @@ export class AppStack extends Stack {
...
+      'datadog-api-key-secret',
+      config.datadogApiKeySecretArn,
+    );
+
+    const datadog = new Datadog(this, 'datadog', {
+      apiKeySecret: datadogSecret,
+      addLayers: false,
+      enableDatadogLogs: false,
+      flushMetricsToLogs: false,
+      extensionLayerVersion: 58,
```
[the remark this answers is not in the document]
Comment (samchungy) [reply template/lambda-sqs-worker-cdk/infra/appStack.ts:85]: II was planning on setting up a customManager in rynovate under `infra/.*ts` to renovate these.

https://github.com/DataDog/datadog-cdk-constructs/issues/232

File: template/lambda-sqs-worker-cdk/src/testing/logging.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    logger.error.mockClear();
+    logger.info.mockClear();
+    logger.debug.mockClear();
+  },
+
+  spy: () => {
+    jest.spyOn(logging.logger, 'error').mockImplementation(logger.error);
+    jest.spyOn(logging.logger, 'info').mockImplementation(logger.info);
+    jest.spyOn(logging.logger, 'debug').mockImplementation(logger.debug);
+  },
```
[the remark this answers is not in the document]
Comment (72636c) [reply template/lambda-sqs-worker-cdk/src/testing/logging.ts:18]: Ref regarding 2: SEEK-Jobs/indie-hirer-posting-preferences-api#44

**Comment by samchungy** [comment]
Awaiting https://github.com/DataDog/datadog-cdk-constructs/pull/303

**Comment by lukeymyuan** [comment]
Can you add README file similar to other templates?

File: template/lambda-sqs-worker-cdk/src/services/jobScorer.ts
[the remark this answers is not in the document]
Comment (samchungy) [reply template/lambda-sqs-worker-cdk/src/services/jobScorer.ts:1]: Think we can address these all in another PR

File: template/lambda-sqs-worker-cdk/infra/appStack.ts [hunk 3]
```diff
@@ -8,14 +8,18 @@ import {
...
   aws_sns,
-  aws_sns_subscriptions,
   aws_sqs,
 } from 'aws-cdk-lib';
 import type { Construct } from 'constructs';
+import { Datadog } from 'datadog-cdk-constructs-v2';
 
 import { config } from './config';
 
+// Updated by https://github.com/seek-oss/rynovate
```
Comment (samchungy) [inline template/lambda-sqs-worker-cdk/infra/appStack.ts:20]: https://github.com/seek-oss/rynovate/pull/137

**Pull request merged**